Swap.cpp

```diff
@@ -0,0 +1,55 @@
+#include<iostream>
+using namespace std;
+
+void SwapRef(int &a, int &b) 
+{
+   int temp;
+   temp = a; 
+   a = b;    
+   b = temp; 
+     
+   return;
```

Comment:
Since its a void function , there is no point explicitly writing a return statement here. It will anyways return when the function is finished :)